README.md

```diff
@@ -30,32 +30,9 @@ session files to force log out all of your users.
 Notes
 -----
 
-This plugin creates a new XMPP session on each page rendering in order to
-support multiple active window. To avoid confusion it is recommended to have
-the XMPP server broadcast incoming messages to all resources.
-
-* Openfire: "route.all-resources: true"
-* Prosody: "ignore_presence_priority = true"
-
-Troubleshooting
----------------
-
-Read [troubleshooting](TROUBLESHOOTING.md) if you are having problem. 
-
-Development
------------
-The plugin has a complete and minified version of the converse.js library with
-all its dependencies included. For development, you can pull converse.js as
-a git submodule and include the scripts and style sheets directly from there.
-
-1. Load the converse.js and xmpp-prebind-php submodule:
-   ```
-   cd cd your_roundcube_dir/plugins/converse
-   git submodule init && git submodule update
-   ```
-
-2. Set `$rcmail_config['converse_xmpp_devel_mode'] = true;` in this plugins
-   config file.
+This plugin using the same resource (both BOSH prebind and active) in order to fix stability, but there is no support for multiple web pages with one jid's s session. Everytime you make one page active will result in close connection on the other page.
+This version uses only login authentication method of conversejs (utilizes jid + password) which is not secure. There is a prebind method that utilizes rid, jid and sid from prebind_url in future plan (code for this implementation is commented now in converse.php).
+Fill free to implement prebind.php for this purpose and make PR ;)
```

Comment:
> This plugin using the same resource (both BOSH prebind and active) in order to fix stability

This doesn't seem necessary. Why can't each tab not have a separate resource and connection? This is how many people run Converse across multiple tabs.

Reply:
I will be fixed in next update.

Reply:
> > This plugin using the same resource (both BOSH prebind and active) in order to fix stability
> 
> This doesn't seem necessary. Why can't each tab not have a separate resource and connection? This is how many people run Converse across multiple tabs.
I will be fixed in the next update.